admin.go

```diff
@@ -87,11 +86,8 @@ func AdminRebuildDBHandler(ctx *handlerContext, w http.ResponseWriter, r *http.R
 
 	//return /*
 	//db.Close()
-	os.Remove("./RebbleAppStore.db")
```

Comment:
Oh, yeah I did forget to add a replacement for that line.

What do you think about dropping the table, rather than removing the entire database? If the entire database is corrupted you can just delete the file manually and the server can re-create it on startup.